src/Notify/Notifier/AppleNewsNotifier.php

```diff
@@ -156,12 +156,42 @@ protected function updateArticle(Message $notification, Message $app, Message $a
         }
 
         $latestRevision = $this->getLatestRevision($notification);
-        if (null === $latestRevision || $article->get('apple_news_revision') === $latestRevision) {
+        if (null !== $latestRevision && $metadata['revision'] !== $latestRevision) {
+            $metadata['revision'] = $latestRevision;
+            $result = $this->api->updateArticle((string)$article->get('apple_news_id'), $document, $metadata);
+        }
+
+        if ($result['ok']) {
+            return $result;
+        }
+
+        $getArticleResult = $this->api->getArticle((string)$article->get('apple_news_id'));
```

Comment:
@tbone1020 use fget instead of adding string casting.

Reply:
done